src/Generators/Microsoft.Gen.Metrics/Parser.cs

```diff
@@ -656,7 +656,7 @@ private void Diag(DiagnosticDescriptor desc, Location? location, params object?[
                 ex.NamedType.ToDisplayString());
         }
 
-        if (strongTypeAttributeParameters.StrongTypeConfigs.Count > MaxTagNames)
+        if (strongTypeAttributeParameters.DimensionHashSet.Count > MaxTagNames)
```

Comment:
```suggestion
        if (strongTypeAttributeParameters.TagHashSet.Count > MaxTagNames)
```

Reply:
The terminology exposed is "Tag" rather than "Dimension", correct?

Reply:
Is it OK now?